apps/publikator/components/editor/modules/author/ui.js

```diff
@@ -33,6 +34,7 @@ export default ({ TYPE, newBlock, editorOptions }) => {
     if (!value.blocks.some(matchBlock(TYPE))) {
       return null
     }
+    const handlerFactory = createOnFieldChange(onChange, value)
```

Comment:
pedantic but for me `handlerFactory` is not a very clear name, except if it's some kind of standard?

Reply:
Yes, I found it as a standard way to create Field onChange Events (see links)